docs/Hardlinks/How-to-setup-for/Windows.md

```diff
@@ -0,0 +1,247 @@
+# Windows
+
+Windows is very limited when it comes to hardlinks/instant moves (Atomic Moves) support. Basically, it comes down to just using one disk for your download location and media library. However, we will try to explain an option that you can make use of hardlinks/instant moves (Atomic Moves) if you run a two-disk setup.
+
+## Recommendation
+
+We would recommend switching to another operating system if you want to make use of a hardlink setup. For more information and suggestions on this topic, we recommend joining our Discord support channel. The link can be found at the bottom of every guide.
+
+### Alternative recommendation
+
+If you have multiple disks and still want to keep using Windows, we suggest forgetting about hardlinks/instant moves (Atomic Moves) and going for the next best, most recommended option for Windows, namely [StableBit DrivePool](https://stablebit.com/){:target="\_blank" rel="noopener noreferrer"}. This will make it possible to make it look like you have one big disk, making it much easier to maintain and set up your file and folder structure.
```

Comment:
```suggestion
If moving away from using Windows with multiple disks is not an option, we suggest ignoring hard links and instant (Atomic) moves and using [StableBit DrivePool](https://stablebit.com/){:target="\_blank" rel="noopener noreferrer"}. This will allow you to pool multiple disks to appear as one big disk, making it much easier to maintain and set up your file and folder structure.
```